### DISCUSSION TITLE
Multiple page (screens) capability

### DISCUSSION THREAD

**Discussion by hranmuthu** [geovens/gInk#175] [Ideas]
This is a fantastic tool. I just fell in love with it from the day I installed it. 

Sometimes we need to draw in more than one page. If we have feature to turn pages (drawing pages) and draw, then we can keep multiple drawings in one go. This way when we change the background app, we can put the correct page on gInk.

**Comment by pubpub-zz** [comment]
@hranmuthu,
you may be interested into having a look at the fork I developed (https://github.com/pubpub-zz/ppInk):
You can configure ppInk  in order to carry on drawing after you've done a snapshot (by default long click exits after and long click on snapshot carry on drawing, but you can invert the two behaviors:
![image](https://user-images.githubusercontent.com/4083478/189720981-e75c7e0b-4e63-4d91-b839-9b522aa8e567.png)  )

also you ppInk has a capability to do some multiple captures in pointer mode (cf "snapshots in pointer mode" in https://github.com/pubpub-zz/ppInk)
once the captures about the application have been captured, when you exit pointer mode to drawing, the snapshots are pasted from the oldest one to the most recent one, and you can add your drawings, do the snapshots (as show above) and to remove the captured screen use the erasor on the border of the image to delete the snapshot and show the previously captured one.
